tests/test_utils.py

```diff
@@ -46,14 +46,49 @@ def test_imresize():
     )
     assert resized_img.shape == (1000, 500, 3)
 
-    # test for dtype conversion
+    # test for dtype conversion, pairs of
+    # (original type, converted type)
+    test_dtypes = [
+        (np.bool, np.uint8),
+        (np.int8, np.int16),
```

Comment:
I understand why you are doing this. Why np.int8 converted to np.int16 ? Is np.int16 not supported in opencv?

Reply:
`np.int8` is not supported but `np.int16` is supported. Hence, `np.int8` (original type) will be converted to `np.int16` (converted type).